jdaviz/configs/specviz/plugins/unit_conversion/unit_conversion.py

```diff
@@ -42,16 +43,21 @@ class UnitConversion(PluginTemplateMixin):
     * :meth:`~jdaviz.core.template_mixin.PluginTemplateMixin.close_in_tray`
     * ``spectral_unit`` (:class:`~jdaviz.core.template_mixin.SelectPluginComponent`):
       Global unit to use for all spectral axes.
-    * ``flux_unit`` (:class:`~jdaviz.core.template_mixin.SelectPluginComponent`):
+    * ``flux_or_sb_unit`` (:class:`~jdaviz.core.template_mixin.SelectPluginComponent`):
```

Comment:
I think this requires some thought/discussion.  Do we want a single select/dropdown for `flux_or_sb_unit` that changes its choices based on `flux_or_sb` _or_ separate `flux_unit` and `sb_unit` and just toggle which is shown - or maybe even show both (and then have changing one automatically update the other to match).  

I'm leaning towards the second option since we'll eventually probably want to bring this to Imviz where there won't be a `flux_or_sb` selection, but will want to have a selection for surface brightness units which will affect the resulting flux units (from aperture photometry, for example).